core/domain/question_services.py

```diff
@@ -152,13 +152,12 @@ def get_questions_by_skill_ids(total_question_count, skill_ids):
 
     Returns:
         list(Question). The list containing an expected number of
-        total_question_count questions linked to each given skill id.
-        It will return more if total_question_count is not evenly divisable,
-        or less if not enough questions in datastore.
+            total_question_count questions linked to each given skill id.
+            question count per skill will be total_question_count divided by
+            length of skill_ids, and it will be rounded up if not evenly
+            divisable. The order of questions will follow the order of given
```

Comment:
divisable --> divisible

This description is good except for one thing. Please explain what happens if there are insufficient questions for a different skill.

> I think the issue of fetching the same question multiple times is slightly out of scope of this PR. In the latter part of this GSoC project, the fetch functions will be changing anyway.

This should be filed as an issue since it impacts the user experience (for the student to receive the same question twice in a session/test).

Reply:
I will file an issue for this (However it will not impact user experience as we are not planning to ship it with this experience right now anyway)

Reply:
Thanks, please add the issue number here once you've done that. (To be clear, I meant that it would impact the UX when this feature is released to students.)

Reply:
I didn't completely understand how it will impact UX because we wont release the feature without the change in the fetch function. Anyway I have added an issue (#6930) to change the fetch function as mentioned in the GSoC project.

Reply:
Thanks for filing the issue, @vinitamurthi! 

Ah sorry, I think this might be a case where you have more insight than I do into the "hotlist" that needs to be completed prior to release. I'd been taking it to be the list of issues in the [Practice Sessions project on GitHub](https://github.com/oppia/oppia/projects/33) (hence the request to file an issue). Maybe we can discuss this when we meet next.